src/Parsers/ASTFunction.cpp

```diff
@@ -270,6 +270,31 @@ void ASTFunction::formatImplWithoutAlias(const FormatSettings & settings, Format
             }
         }
 
+        if (!written && arguments->children.size() == 1)
```

Comment:
Let's add a comment that this is for postfix unary operators.

Reply:
Done